python_modules/dagster/dagster_tests/cli_tests/command_tests/test_telemetry.py

```diff
@@ -570,13 +566,12 @@ def asset2():
         repository_handle=MagicMock(spec=RepositoryHandle),
     )
     stats = get_stats_from_external_repo(external_repo)
-    assert (
-        stats["dagster_resources"]
-        == "[{'module_name': 'dagster_tests', 'class_name': 'MyIOManager'}, {'module_name':"
-        " 'dagster_tests', 'class_name': 'my_io_manager'}, {'module_name': 'dagster_tests',"
-        " 'class_name': 'my_resource'}, {'module_name': 'dagster_tests', 'class_name':"
-        " 'MyResource'}]"
-    )
+    assert stats["dagster_resources"] == [
+        {"module_name": "dagster_tests", "class_name": "MyIOManager"},
+        {"module_name": "dagster_tests", "class_name": "my_io_manager"},
+        {"module_name": "dagster_tests", "class_name": "my_resource"},
+        {"module_name": "dagster_tests", "class_name": "MyResource"},
+    ]
     assert stats["has_custom_resources"] == "False"
 
 
```

Comment:
What i'd really like to do is add some kind of test where we `json.loads` a telemetry log with metadata and ensure that it loads correctly, to get some confidence that the parsing in purina/snowflake will work. but i'm having a hard time getting a telemetry log with the metadata info in it.

Reply:
fwiw I think we basically just need to json.loads the metadata itself, which can probably be done in pure python?

Reply:
the issue i was having with that is that at this stage, `stats` is json loadable even with the old bad code. so i think theres something with writing and reading the string that's maybe making a difference? i should look into it and figure out the specifics